app/src/main/java/com/yello/presentation/main/recommend/RecommendInviteDialog.kt

```diff
@@ -0,0 +1,84 @@
+package com.yello.presentation.main.recommend
+
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import com.example.ui.base.BindingDialogFragment
+import com.example.ui.fragment.snackBar
+import com.example.ui.fragment.toast
+import com.example.ui.view.setOnSingleClickListener
+import com.kakao.sdk.common.util.KakaoCustomTabsClient
+import com.kakao.sdk.share.ShareClient
+import com.kakao.sdk.share.WebSharerClient
+import com.yello.R
+import com.yello.databinding.FragmentRecommendInviteDialogBinding
+import timber.log.Timber
+
+class RecommendInviteDialog :
+    BindingDialogFragment<FragmentRecommendInviteDialogBinding>(R.layout.fragment_recommend_invite_dialog) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.btnInviteDialogExit.setOnSingleClickListener {
+            dismiss()
+        }
+
+        binding.btnInviteKakao.setOnSingleClickListener {
+            startKakaoInvite(this.requireContext())
+        }
+
+        binding.btnInviteLink.setOnSingleClickListener {
+            startLinkInvite()
+        }
+    }
+
+    // TODO : 스낵바 커스텀 & 링크 복사
+    private fun startLinkInvite() {
+        requireParentFragment().snackBar(requireParentFragment().requireView()) { "링크가 복사되었습니다." }
+        requireParentFragment().toast("링크가 복사되었습니다.")
```

Comment:
```suggestion
        binding.btnInviteKakao.setOnSingleClickListener {
            startKakaoInvite(requireContext())
        }

        binding.btnInviteLink.setOnSingleClickListener {
            startLinkInvite()
        }
    }

    // TODO : 스낵바 커스텀 & 링크 복사
    private fun startLinkInvite() {
        requireParentFragment().snackBar(requireParentFragment().requireView()) { "링크가 복사되었습니다." }
        toast("링크가 복사되었습니다.")
```
수신 객체 생략 가능할 것 같아요!
"링크가 복사되었습니다" 얘도 strings.xml에 추가해서 사용하는 것이 좋을 것 같아요! getString() 함수 활용하시면 됩니당